Sources/Wendy/cli/commands/RunCommand.swift

```diff
@@ -444,24 +462,32 @@ extension RunCommand {
                                 "Uploading layer to agent",
                                 metadata: ["digest": .string(layer.digest)]
                             )
-                            try await agentContainers.writeLayer(
-                                request: .init { writer in
-                                    try await layer.withStream { chunk in
-                                        try await writer.write(
-                                            .with {
-                                                $0.digest = layer.digest
-                                                $0.data = Data(chunk)
-                                            }
-                                        )
+                            do {
+                                try await agentContainers.writeLayer(
+                                    request: .init { writer in
+                                        try await layer.withStream { chunk in
+                                            try await writer.write(
+                                                .with {
+                                                    $0.digest = layer.digest
+                                                    $0.data = Data(chunk)
+                                                }
+                                            )
+                                        }
+                                    }
+                                ) { response in
+                                    for try await _ in response.messages {
+                                        // Ignore responses
```

Comment:
Any reason we ignore responses here? Do the messages just not have any useful information in this context?

Reply:
We're not currently sending back anything during layer upload. If anything I could warn here, since we don't know the message type yet